packages/core/src/components/popover2/popover2.tsx

```diff
@@ -167,11 +161,13 @@ export interface IPopover2Props extends IOverlayableProps, IProps {
 
     /**
      * The position (relative to the target) at which the popover should appear.
-     * The default value of `"auto"` will choose the best placement when opened and will allow
-     * the popover to reposition itself to remain onscreen as the user scrolls around.
+     *
+     * The default value of `"auto"` will choose the best position when opened
+     * and will allow the popover to reposition itself to remain onscreen as the
+     * user scrolls around.
      * @default "auto"
      */
-    placement?: Placement;
+    position?: Position | "auto";
```

Comment:
does it make sense to add `"auto"` in `Position`?

Reply:
meh `auto` is only supported by Popover, whereas `Position` is used in many other places.